### ISSUE TITLE
bring into LINUX client the features from Windows client

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#14]
lack of features on linux as the windows client
* no participation in superblock
* no voting

this is on the [roadmap](https://steemit.com/gridcoin/@erkan/gridcoin-roadmap-2017-2019) (2017 June/July)

mentioned in [hangout 1](https://soundcloud.com/gridcoin-community-hangouts/gridcoin-hangout-001): 0:40:00 - 0:42:00

**Comment by grctest** [comment]
Rob has been discussing this recently, and it has been included in the roadmap.

**Comment by TG13088** [comment]
Ideally the UI should be Operating System neutral and uniform across all implementations.  This will lead to less confusion for the new user and less aggravation for the Help/Support people that answer their questions.  It should be a small matter to *grey out* items that don't pertain to the operating environment it is being hosted under.

**Comment by grctest** [comment]
> @caraka [wrote](https://github.com/gridcoin/Gridcoin-Research/pull/233#issuecomment-294572979):
> The 'windows first' attitude is exactly why we have so many problems with the neural network. For the reputation of gridcoin, Rob, team, etc, we should focus on good c++ code that works on ALL machines, the way that it was intended. Why? because windows isn't nearly as common among gridcoin users as people would think. Because coding for windows and then 'back filling' is nonsense when the language you are coding in is universal. The sooner we lose the .net code bolted into gridcoin, the better off we'll be. We're more stable with diversity, not a monoculture.

I agree that getting all functionality to c++ is important, there are many linux users within the Gridcoin network. We should place a bounty on this development task.

Related tasks: 
https://github.com/gridcoin/Gridcoin-Research/issues/212
https://github.com/Erkan-Yilmaz/Gridcoin-tasks/issues/94
